### DISCUSSION TITLE
Inserting/removing canvas space

### DISCUSSION THREAD

**Discussion by rockinator** [jgraph/drawio#3950] [Ideas]
I would love to have a feature to insert and remove canvas space, either horizontically or vertically, see GIF below with an example of what I mean. This is great when modelling processing and needing to add or remove a process step.
![Untitled video - Made with Clipchamp](https://github.com/jgraph/drawio/assets/22370880/ab3ed1fb-cb7c-43ad-963e-b69c43c9d66a)

**Comment by pzatschl** [comment]
Don't know, if it helps you, but there is Alt+Ctrl+Shift+Drag (which isn't locked to an axis afaik). I have asked a question on this here: #4419
